cfme/tests/cloud_infra_common/test_retirement.py

```diff
@@ -84,7 +83,7 @@ def verify_retirement_state(retire_vm):
     """
     # wait for the info block showing a date as retired date
     # Use lambda for is_retired since its a property
-    view_cls = navigator.get_class(retire_vm, 'Details')
+    view_cls = navigator.get_class(retire_vm, 'Details').VIEW
```

Comment:
This is great! Wasn't actually aware this method existed, and it will definitely come in handy.